doc/build-windows.md

```diff
@@ -62,4 +62,3 @@ To build executables for Windows 64-bit:
     ./configure --prefix=`pwd`/depends/x86_64-w64-mingw32
     make
 
-For further documentation on the depends system see [README.md](../depends/README.md) in the depends directory.
```

Comment:
why you have removed this line. `depends/README.md` indeed contains further documentation on the depends system. This is valid for BU and Core. Am I missing something obvious?